src/core/ledger/ledger.js

```diff
@@ -768,6 +774,79 @@ export class Ledger {
     account.payoutAddresses.delete(JSON.stringify(currencyResult.value));
   }
 
+  /**
+   * no-ops if integration tracking is already enabled
+   */
+  enableIntegrationTracking(): Ledger {
+    if (!this._trackIntegration) {
+      this._createAndProcessEvent({
+        type: "TOGGLE_INTEGRATION",
```

Comment:
We've kinda regretted using TOGGLE for identity activations, because it makes it really hard to reason about whether it's a toggle on or off. Would probably be better to learn from that regret and differentiate between ENABLE_ and DISABLE_ ledger events. Also, "integration" is kinda vague, and it'd be nice to be more specific like ENABLE_PAYOUT_INTEGRATION or ENABLE_DISTRIBUTION_INTEGRATION or ENABLE_GRAIN_INTEGRATION and then use the same terminology for the method name like "enablePayoutTracking" or whatever.